node/test_helpers.go

```diff
@@ -1,16 +1,70 @@
 package node
 
 import (
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	testutils "github.com/celestiaorg/utils/test"
 	"github.com/libp2p/go-libp2p/core/crypto"
+	"github.com/rollkit/rollkit/conv"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"github.com/tendermint/tendermint/crypto/ed25519"
 	"github.com/tendermint/tendermint/p2p"
 	tmtypes "github.com/tendermint/tendermint/types"
-
-	"github.com/rollkit/rollkit/conv"
 )
 
 var genesisValidatorKey = ed25519.GenPrivKey()
 
+type MockTester struct {
+	t *testing.T
+}
+
+func (m MockTester) Fail() {
+	fmt.Println("Failed")
+}
+
+func (m MockTester) FailNow() {
+	fmt.Println("MockTester FailNow called")
+}
+
+func (m MockTester) Logf(format string, args ...interface{}) {
+	fmt.Println("MockTester Logf called")
+}
+
+func (m MockTester) Errorf(format string, args ...interface{}) {
+	//fmt.Printf(format, args...)
+	fmt.Println("Errorf called")
+}
+
+func waitForFirstBlock(assert *assert.Assertions, require *require.Assertions, node Node) {
+	require.NoError(testutils.Retry(300, 100*time.Millisecond, func() error {
+		fn, ok := node.(*FullNode)
+		assert.Equal(ok, true)
+		if fn.Store.Height() >= 1 {
+			return nil
+		} else {
+			fmt.Println("Retrying...")
+			return errors.New("awaiting first block")
```

Comment:
I would make this errors as if they were the last error. So making them descriptive as to the state of the failure, what height the node is at is more helpful for debugging.